tinyman/v1/pools.py

```diff
@@ -284,11 +284,12 @@ def convert(self, amount: AssetAmount):
             return AssetAmount(self.asset1, int(amount.amount * self.asset2_price))
 
     def fetch_mint_quote(
-        self, amount_a: AssetAmount, amount_b: AssetAmount = None, slippage=0.05
+        self, amount_a: AssetAmount, amount_b: AssetAmount = None, slippage=0.05, refresh=True
```

Comment:
Optional: Should we add the type hint for the `refresh` parameters?

Ex:
`refresh=True` -> `refresh: bool = True`

Reply:
Hmm I think it does default to bool if the default value is a bool. Similar how `slippage` defaults to float since the default value is a float (0.05). Up to you, idc either way. Or maybe `black` has a rule for that?